packages/node-sdk/README.md

```diff
@@ -0,0 +1,223 @@
+# Bucket Node.js SDK
+
+Node.js, JavaScriptS/Typescript feature flag and tracking client for [Bucket.co](https://bucket.co).
+
+## Installation
+
+Install using `yarn` or `npm` with:
+
+> `yarn add -s @bucketco/node-sdk` or `npm install -s @bucketco/node-sdk`.
+
+Other languages than JavaScript/Typescript are currently not supported by an SDK.
+You can [use the HTTP API directly](https://docs.bucket.co/reference/http-tracking-api)
+
+## Basic usage
+
+Before the library can be used, you need to obtain a secret key from
+[Environment setting view](https://app.bucket.co/envs/{environment}/settings/app-environments)
+in **Bucket.co**.
+
+> [!CAUTION]
+> Secret keys should only be used by the `node-sdk` and not any other
+> SDKs provided by Bucket. Secret keys offer the users the ability to obtain
+> information that is often sensitive and thus should not be used in any
+> front-end application.
+
+```ts
+import { BucketClient } from "@bucketco/node-sdk";
+
+// Create a new instance of the client with the secret key. Additional options
+// are available, such as supplying a logging sink, a custom HTTP client and
+// other custom properties.
+//
+// We recommend that only one global instance of `client` should be created
+// to avoid multiple round-trips to our servers.
+const client = new BucketClient({
+  secretKey: "sec_prod_xxxxxxxxxxxxxxxxxxxxx",
+});
+
+// Initialize the client's feature flag retrieval loop. You must call this method
+// prior to any calls to `getFlags()`, otherwise an empty object will be returned.
+//
+// You can also supply your `fallbackFlags` to the `initialize()` method. These
+// fallback flags are used in the situation when the server-side flags are not
+// yet loaded or there are issues retrieving them.
+await client.initialize();
```

Comment:
```suggestion
// Initialize the client and begin fetching flag definitions.
// You must call this method prior to any calls to `getFlags()`,
// otherwise an empty object will be returned.
//
// You can also supply your `fallbackFlags` to the `initialize()` method. These
// fallback flags are used in the situation when the server-side flags are not
// yet loaded or there are issues retrieving them.
// See "Initialization Options" below for more information.
await client.initialize();
```